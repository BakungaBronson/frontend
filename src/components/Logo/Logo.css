.HeaderLogo {
  margin: 3rem;
  width: 50%;
}