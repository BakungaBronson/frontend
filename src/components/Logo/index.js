import React from 'react';
// import { Link } from 'react-router-dom';
import { ReactComponent as LogoIcon } from '../../assets/images/secondary-logo-inverted.svg';
import './Logo.css';

const Logo = () => {
  return (
    // <Link to="/">
    <LogoIcon className="Logo" />
    // </Link>
  );
};

export default Logo;
