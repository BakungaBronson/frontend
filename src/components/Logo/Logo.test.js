import React from 'react';

import { shallow } from 'enzyme';

import Logo from './index';

describe('Logo Component', () => {
    it('Renders a Logo component', () => {
        const LogoComponent = shallow(<Logo />);
        
        expect(LogoComponent).toMatchSnapshot();

        const LogoType = LogoComponent.find("div > div");
        expect(LogoType.hasClass("LogoType")).toEqual(true);
        expect(LogoType.text()).toBe("Crane Cloud");
    })
})