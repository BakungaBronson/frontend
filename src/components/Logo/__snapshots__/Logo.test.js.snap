// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component Renders a Logo component 1`] = `ShallowWrapper {}`;
